app/controllers/api/v1/reservations.rb

```diff
@@ -0,0 +1,43 @@
+class Api::V1::ReservationsController < ApplicationController
+  before_action :set_reservation, only: %i[create]
+  before_action :set_user, only: %i[update destroy]
+  before_action :authorized
```

Comment:
I couldn't see this function anywhere in the codebase. You can rest assured that only authorized users will be able to access this endpoint so this line may be unnecessary

Reply:
Hi @Shisui6 
Which line are you refering to?

Reply:
hey @Sonickmumba , line 4